code/modules/martial_arts/muscle_implant.dm

```diff
@@ -0,0 +1,55 @@
+/datum/martial_art/muscle_implant
+	name = "Empowered Muscle Implant Fighting"
+	weight = 1
+	/// Are we EMP'ed?
+	var/is_emp
+	/// How much damage should our punches deal
+	var/punch_damage = 13
+	var/picked_hit_type
```

Comment:
Can you remove this from a var on the datum, and just pick randomly in do_the_punch

Reply:
It also uses this in the main attack proc for the shield stuff

Reply:
Okay, then it can still be picked, then just transfered to do_the_punch through an argument, or just have 2 entirely different pick() calls